code/modules/admin/outfit_editor.dm

```diff
@@ -0,0 +1,187 @@
+/client/proc/open_outfit_editor(datum/outfit/target)
+	var/datum/outfit_editor/ui = new(usr, target)
+	ui.ui_interact(usr)
+
+
+#define OUTFIT_EDITOR_NAME "Outfit-O-Tron 9000"
+/datum/outfit_editor
+	var/client/holder
+
+	var/dummy_key
+
+	var/datum/outfit/drip
+
+/datum/outfit_editor/New(user, datum/outfit/target)
+	holder = CLIENT_FROM_VAR(user)
+
+	if(ispath(target))
+		drip = new /datum/outfit
+		drip.copy_from(new target)
+	else if(istype(target))
+		drip = target
+	else
+		drip = new /datum/outfit
+		drip.name = "New Outfit"
+
+/datum/outfit_editor/ui_state(mob/user)
+	return GLOB.admin_state
+
+/datum/outfit_editor/ui_status(mob/user, datum/ui_state/state)
+	if(QDELETED(drip))
+		return UI_CLOSE
+	return ..()
+
+/datum/outfit_editor/ui_close(mob/user)
+	clear_human_dummy(dummy_key)
+	qdel(src)
+
+/datum/outfit_editor/proc/init_dummy()
```

Comment:
This is the second time I've used dummies in pretty much the same way, maybe I should make some centralized way of using dummy lookalikes. Not in this PR though.
By the way I haven't forgotten about the `equip` problem pointed out in https://github.com/tgstation/tgstation/issues/57865